custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -27,22 +27,25 @@ def aggregation_query(self):
           SELECT
             %(state_id)s AS state_id,
             %(month)s AS month,
-            awc_id,
-            block_id,
-            state_name,
-            district_name,
-            block_name,
-            supervisor_name,
-            awc_name,
-            aww_name,
-            contact_phone_number,
-            wer_weighed,
-            wer_eligible,
-            awc_num_open,
-            valid_visits,
-            expected_visits
-          FROM agg_ccs_record_monthly AS acm
-          WHERE acm.month = %(month)s AND acm.state_id = %(state_id)s and aggregation_level=5
+            awcm.awc_id,
+            awcm.block_id,
+            awcm.state_name,
+            awcm.district_name,
+            awcm.block_name,
+            awcm.supervisor_name,
+            awcm.awc_name,
+            awcm.aww_name,
+            awcm.contact_phone_number,
+            awcm.wer_weighed,
+            awcm.wer_eligible,
+            awcm.awc_num_open,
+            sum(ccsm.valid_visits),
+            sum(ccsm.expected_visits)
+          FROM agg_awc_monthly as awcm
+          INNER JOIN agg_ccs_record_monthly AS ccsm
```

Comment:
not related to this PR but I'm curious to know why it's necessary to use these 'monthly' views. These views do a cross join with the months table which is impossible in a sharded environment.

Reply:
I used those views because they made it easy to get the location info (particularly the names). If the only issue is the months table, that would be an easy fix, but I would still need a join between the `awc_location` table and the agg tables I use